include/nektar_interface/particle_cell_mapping/generated_linear/hexahedron.hpp

```diff
@@ -0,0 +1,277 @@
+/**
```

Comment:
Can we modify the python so that it writes a comment in the generated file saying which python file(s) it came from? Or even how it was generated `python foo.py --arg1 bar --arg2 baz` something like that?